ios/gutenberg/GutenbergViewController.swift

```diff
@@ -35,3 +37,53 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         callback("https://cldup.com/cXyG__fTLN.jpg")
     }
 }
+
+//MARK: - Navigation bar
+
+extension GutenbergViewController {
+    
+    func configureNavigationBar() {
+        addSaveButton()
+        addMoreButton()
+    }
+    
+    func addSaveButton() {
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .save,
+                                                           target: self,
+                                                           action: #selector(saveButtonPressed(sender:)))
+    }
+    
+    func addMoreButton() {
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "...",
+                                                            style: .plain,
+                                                            target: self,
+                                                            action: #selector(moreButtonPressed(sender:)))
+    }
+}
+
+//MARK: - More actions
+
+extension GutenbergViewController {
+    
+    func showMoreSheet() {
+        let alert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        
+        let toggleHTMLModeAction = UIAlertAction(
+            title: htmlMode ? "Switch To Visual" : "Switch to HTML",
+            style: .default,
+            handler: { [unowned self] action in
+                self.toggleHTMLMode(action)
+            }
+        )
+        let cancelAction = UIAlertAction(title: "Keep Editing", style: .cancel)
+        alert.addAction(toggleHTMLModeAction)
+        alert.addAction(cancelAction)
+        
+        present(alert, animated: true)
+    }
+    
+    func toggleHTMLMode(_ action: UIAlertAction) {
+        self.htmlMode = !self.htmlMode
+        self.gutenberg.toggleHTMLMode()
```

Comment:
I don't think you need any of the `self.` here? So much JS lately I'm not even sure about Swift anymore 😱

Reply:
no you are right, this should be a result of a copy paste from aztec side of things